src/graphql/queries/hooks/patients.ts

```diff
@@ -1,11 +1,17 @@
+import React from 'react';
 import { useQuery } from '@apollo/react-hooks';
 import { useApolloClient } from 'react-apollo';
 import { GET_ALL_PATIENTS } from '../patients';
 
-export function useAllPatients() {
-  const { data } = useQuery(GET_ALL_PATIENTS);
+export function useAllPatients(eventId?: string, connectionData?: string) {
+  const { data, refetch } = useQuery(GET_ALL_PATIENTS);
   const client = useApolloClient();
 
+  React.useEffect(() => {
+    // fetches from backend when eventId changes
+    if (connectionData === 'connected') refetch();
```

Comment:
if `connectionData` is just a string, we can just call it `connectionStatus`
```suggestion
    if (connectionStatus === 'connected') refetch();
```

Reply:
or is it a typo? i notice that we're calling `connectionData.networkStatus` in the ccp dashboard

Reply:
my bad, this is a typo ie. i passed in the wrong parameter. will name the var for better clarity